tools/realsense-viewer/realsense-viewer.cpp

```diff
@@ -440,20 +440,36 @@ int main(int argc, const char** argv) try
         if (ImGui::BeginPopup("select"))
         {
             ImGui::PushStyleColor(ImGuiCol_Text, dark_grey);
-            ImGui::Columns(2, "DevicesList", false);
+            ImGui::Columns(1, "DevicesList", false);
             for (int i = 0; i < device_names.size(); i++)
             {
                 bool skip = false;
                 for (auto&& dev_model : *device_models)
                     if (get_device_name(dev_model->dev) == device_names[i]) skip = true;
                 if (skip) continue;
 
+                auto dev = connected_devs[i];
+                std::string dev_type;
+                if( dev.supports( RS2_CAMERA_INFO_USB_TYPE_DESCRIPTOR ) )
+                {
+                    dev_type = "USB";
+                    dev_type += dev.get_info( RS2_CAMERA_INFO_USB_TYPE_DESCRIPTOR );
+                }
+                else if( dev.supports( RS2_CAMERA_INFO_PRODUCT_ID ) )
+                {
+                    dev_type = dev.get_info( RS2_CAMERA_INFO_PRODUCT_ID );
```

Comment:
The Viewer can now show multiple device types (if there are multiple).
In some situations, it can even show the *same* device available multiple ways (one USB, one DDS).
I wanted the device selection to make it clear which are USB, which DDS, and which GMSL.
Those are the only three types right now. And the DDS PID is `DDS`, so it's still the device-type.